mlflow/server/js/src/components/artifact-view-components/ShowArtifactNbView.js

```diff
@@ -0,0 +1,210 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { getSrc } from './ShowArtifactPage';
+import './ShowArtifactNbView.css';
+import { getRequestHeaders } from '../../setupAjaxHeaders';
+import { parseNotebook, fromJS } from "@nteract/commutable";
+import {
+    LightTheme, Cell, Input, Prompt, Source, Outputs, Cells
+} from "@nteract/presentational-components";
+import {
+    Output, StreamText, DisplayData, Media, KernelOutputError, ExecuteResult
+} from "@nteract/outputs";
+import Markdown from "@nteract/markdown";
+import { Provider as MathJaxProvider } from "@nteract/mathjax";
+
+
+class ShowArtifactNbView extends Component {
```

Comment:
What about `ArtifactNotebookView`?

Reply:
I wanted to keep the same syntax as `ShowArtifactTextView` and `ShowArtifactImageView`. Not sure to make this change in this pr, but I prefer `NotebookArtifactView`.